core/payment-driver/zksync/src/driver.rs

```diff
@@ -128,8 +128,9 @@ impl PaymentDriver for ZksyncDriver {
         _caller: String,
         msg: Exit,
     ) -> Result<String, GenericError> {
-        log::info!("EXIT = Not Implemented: {:?}", msg);
-        Ok("NOT_IMPLEMENTED".to_string())
+        wallet::exit(_caller, &msg)
+            .await
+            .and(Ok("Success!".to_string()))
```

Comment:
since `exit()` returns a result you can unwrap it with `?`

I would also make it one line one thought:

```
let tx_hash = wallet::exit(_caller, &msg).await?;
Ok(tx_hash)
```

This requires exit to return the `tx_hash` or `tx_info`